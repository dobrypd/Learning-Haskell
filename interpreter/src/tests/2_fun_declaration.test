int a() {
    return;
}
int b() {
    return 1;
}
bool c() {
    return;
}
