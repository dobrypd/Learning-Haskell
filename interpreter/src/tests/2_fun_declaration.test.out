Welcome to Kappa interpreter
"Function main does not exist!"
[(fromList [],fromList [(Ident "a",([],[SStm (JumpS SjumpFour)])),(Ident "b",([],[SStm (JumpS (SjumpFive (Econst (Eint 1))))])),(Ident "c",([],[SStm (JumpS SjumpFour)]))],fromList [])]
